db/migrations/campaign/V4.0.0.4__table_campaigns.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS campaign.beneficiaries (
+    id uuid NOT NULL DEFAULT (gen_random_uuid()),
+    name character varying(100) NOT NULL,
+    type integer NOT NULL,
+    iso2country_code integer NOT NULL,
+    city character varying(50) NOT NULL,
+    email character varying(100) NULL,
+    phone character varying(50) NULL,
+    website character varying(500) NULL,
+	
+    CONSTRAINT "PK_Beneficiaries" PRIMARY KEY (id)
+);
+
+ALTER TABLE campaign.campaigns
+ADD CONSTRAINT "FK_Campaigns_Beneficiaries_BeneficiaryId" FOREIGN KEY (beneficiary_id) REFERENCES campaign.beneficiaries (id) ON DELETE CASCADE;
```

Comment:
We can add `unique` contstraint to the `email` column so we don't have duplicate beneficiary records for the same person